src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -126,6 +120,8 @@ SurgeGUIEditor::SurgeGUIEditor(void* effect, SurgeSynthesizer* synth) : super(ef
 
    // ToolTipWnd = 0;
 
+   // Before we do much of anything else install our message handlers
```

Comment:
This comment should go; I will just revert this file back to master since it ended up unchanged at the end of all my commits. Sorry!